### PR TITLE
add default parameters for config

### PR DESCRIPTION
**Pull request by RainFallsSilent**

### CODE REVIEW

File: common/config/params.go [hunk 1]
```diff
@@ -171,10 +187,24 @@ func (p *Params) RegNet() *Params {
 		{pk11, "127.0.0.1"},
 		{pk12, "127.0.0.1"},
 	}
+
+	copy.PowLimit = powLimit
```
Comment (alexpan0610) [inline common/config/params.go]: Same values no need to assignment agin.

File: common/config/params.go [hunk 2]
```diff
@@ -114,10 +116,23 @@ func (p *Params) TestNet() *Params {
 		"03dd66833d28bac530ca80af0efbfc2ec43b4b87504a41ab4946702254e7f48961",
 		"02c8a87c076112a1b344633184673cfb0bb6bce1aca28c78986a7b1047d257a448",
 	}
+	copy.PowLimit = powLimit
```
Comment (alexpan0610) [inline common/config/params.go]: Same values no need to assignment agin.

File: common/config/params.go (hunk 1 above)
Comment (RainFallsSilent) [reply common/config/params.go]: fixed

File: common/config/params.go (hunk 2 above)
Comment (RainFallsSilent) [reply common/config/params.go]: fixed

**Pull request merged**